### ISSUE TITLE
[a11y] Important options (resources, DwCA exporting, repairing) missing

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1183]
**Describe the bug**
Update Feed Now, Make DwCA, Resources, and Repair Tree are missing from the user tools dialog.
_Repair tree is debatable, but the other 3 are important._

**To Reproduce**
1. Click on the username
2. See options

**Expected behavior**
These options are important and should be accessible.

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/158605260-0af44824-7851-4ea7-95f9-618be7fb289a.png)
![image](https://user-images.githubusercontent.com/37256050/158605335-897c9127-a58a-4110-9129-e40c90f98ffb.png)

**Comment by grantfitzsimmons** [comment]
Switching collections corrects the issue until you refresh and enter a new session.

**Comment by maxxxxxdlp** [comment]
Fixed by moving to the new permission system.
The User Tool items would now show up if the user has associated permissions
